draft-ietf-tls-dtls-connection-id.md

```diff
@@ -421,13 +448,16 @@ An on-path adversary, who is able to observe the DTLS protocol exchanges between
 DTLS client and the DTLS server, is able to link the observed payloads to all
 subsequent payloads carrying the same connection id pair (for bi-directional
```

Comment:
```suggestion
subsequent payloads carrying the same CID pair (for bi-directional
```

Reply:
Good catch on the CID/connection id terminology